packages/expo-firebase-analytics/src/parseConfig.ts

```diff
@@ -0,0 +1,120 @@
+import { Platform } from 'react-native';
+
+const isAndroid = Platform.OS === 'android';
+
+function isString(str?: string): boolean {
+  return typeof str === 'string';
+}
+function isValidString(str?: string): boolean {
+  return isString(str) && str !== '';
+}
+
+export function isObject(item: any): boolean {
+  return item ? typeof item === 'object' && !Array.isArray(item) && item !== null : false;
```

Comment:
```suggestion
  return item ? typeof item === 'object' && !Array.isArray(item) : false;
```
we check nullability with `item ? … : false`